token/js/test/e2e-2022/transferFee.test.ts

```diff
@@ -32,7 +33,9 @@ const MINT_EXTENSIONS = [ExtensionType.TransferFeeConfig];
 const MINT_AMOUNT = BigInt(1_000_000_000);
 const TRANSFER_AMOUNT = BigInt(1_000_000);
 const FEE_BASIS_POINTS = 100;
+const UPDATED_FEE_BASIS_POINTS= 150;
 const MAX_FEE = BigInt(100_000);
+const UPDATED_MAX_FEE = BigInt(150_000);
```

Comment:
I think since these are only used once in the one test for `setTransferFee`, it's fine to just use them within the test, rather than as top-level constants.